CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 Changelog
 =========
 
-## [Unreleased]
-- No changes yet
+# 0.19.1 (2021-04-02)
+* Fix byte parsing to respect thrift spec.
```

Comment:
Can we provide a little more detail here? E.g., "Fix byte parsing to allow signed integers to match the Thrift spec & other language implementations" or something?